project/src/common/const.ts

```diff
@@ -1,5 +1,30 @@
 const MAX_PERCENTAGE = 100 as const;
 const MAX_OFFER_RATING = 5 as const;
+const MIN_COMMENT_LENGTH = 50;
+const MAX_COMMENT_LENGTH = 300;
+
+const Ratings = [
```

Comment:
```suggestion
const ratings = [
```
нету причин быть с большой буквы)